package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "activemq-artemis-jolokia-api-server",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "https://github.com/artemiscloud/activemq-artemis-jolokia-api-server#readme",
+  "bugs": {
+    "url": "https://github.com/artemiscloud/activemq-artemis-jolokia-api-server/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com:artemiscloud/activemq-activemq-artemis-jolokia-api-server.git"
+  },
+  "scripts": {
+    "build": "yarn clean && tsc -p tsconfig.json && yarn copy-config",
+    "build-api-doc": "yarn openapi-to-md src/config/openapi.yml api.md && yarn pretty-quick",
+    "clean": "rm -rf dist",
+    "copy-config": "cp -r src/config dist/config",
+    "format:all": "prettier --config ./.prettierrc.yml --write './src/**/*'",
+    "lint": "yarn eslint src --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
+    "prepare": "husky install",
+    "start": "node ./dist/app.js",
+    "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 TZ=UTC jest --runInBand",
+    "test:coverage": "yarn run test --watch=false --coverage",
+    "test:generate-output": "yarn test -- --json --outputFile=.jest-test-results.json",
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'"
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "yarn run pretty-quick --staged",
+      "eslint --fix"
+    ]
+  },
+  "devDependencies": {
+    "@types/base-64": "^1.0.2",
+    "@types/cors": "^2.8.17",
+    "@types/express": "^4.17.21",
+    "@types/jest": "27.5.2",
+    "@types/js-yaml": "4.0.5",
+    "@types/jsonwebtoken": "^9.0.6",
+    "@types/node": "^20.10.4",
+    "@types/node-fetch": "^2.6.11",
+    "@types/webpack": "5.28.1",
+    "@typescript-eslint/eslint-plugin": "^5.14.0",
+    "@typescript-eslint/parser": "^5.14.0",
+    "base-64": "^1.0.0",
+    "chromatic": "6.10.1",
+    "copy-webpack-plugin": "11.0.0",
+    "cors": "^2.8.5",
+    "dotenv": "^16.4.5",
+    "eslint": "^8.10.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-unused-imports": "^3.2.0",
+    "express": "4.18.2",
+    "express-openapi-validator": "5.1.2",
+    "express-rate-limit": "^7.2.0",
+    "husky": "8.0.0",
+    "jest": "^27.5.1",
+    "jest-environment-jsdom": "^27.5.1",
+    "js-yaml": "4.1.0",
+    "jsonwebtoken": "^9.0.2",
+    "lint-staged": "13.1.0",
+    "nock": "^13.5.4",
+    "node-fetch": "2",
+    "openapi-to-md": "^1.0.24",
+    "pluralize": "^8.0.0",
+    "prettier": "^2.7.1",
+    "prettier-stylelint": "^0.4.2",
+    "pretty-quick": "3.1.3",
+    "sort-package-json": "^2.10.0",
+    "swagger-routes-express": "^3.3.2",
+    "ts-jest": "^29.2.1",
+    "ts-loader": "^9.3.1",
+    "ts-node": "10.9.2",
+    "typescript": "^4.7.4",
+    "yaml": "^2.4.5"
+  },
+  "consolePlugin": {
```

Comment:
do we need this `consolePlugin`?

Reply:
No we don't, thanks for catching these leftovers